crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/builder/ContributionDTOBuilder.java

```diff
@@ -21,7 +22,7 @@ public static CalculateContributionDTO build(final ApiMaatCalculateContributionR
                 .repId(request.getRepId())
                 .effectiveDate(DateUtil.parseLocalDate(request.getEffectiveDate()))
                 .contributionCap(request.getContributionCap())
-                .monthlyContributions(request.getMonthlyContributions())
+                .monthlyContributions(request.getMonthlyContributions() !=null ? request.getMonthlyContributions() : BigDecimal.ZERO)
                 .upfrontContributions(request.getUpfrontContributions())
```

Comment:
Can we add this check for upfront contributions too? :)

Reply:
Done now